src/clj/puppetlabs/services/versioned_code_service/versioned_code_service.clj

```diff
@@ -21,4 +21,12 @@
    [this environment]
    (if-let [code-id-script (get-in-config [:versioned-code :code-id-command])]
      (vc-core/execute-code-id-script! code-id-script environment)
-     nil)))
\ No newline at end of file
+     nil))
+
+  (get-code-content
+   [this environment code-id file-path]
+   (if-let [code-content-script (get-in-config [:versioned-code :code-content-command])]
```

Comment:
Would it be worth logging a message at init/startup as was done with code-id-command if code-content-command isn't set?